src/fetchServices.ts

```diff
@@ -167,3 +174,32 @@ export async function exportApisToSheets() {
     },
   });
 }
+
+/**
+ * Allow this file to be run locally.
+ * User may enter a single service shortname (e.g. 'spanner') to print the following details to screen:
+ * Service name: spanner.googleapis.com
+ * Service title: Cloud Spanner API
+ * Cloud API: true
+ * Terms of service: serviceusage.googleapis.com/tos/cloud, ...
+ */
+if (require.main === module) {
+  const rl = readline.createInterface({
```

Comment:
So generally if you want to get this fancy, this is where I'd advocate for using `cli.ts` and using the built in argument parser we use over there (meow).  This is just a little weird to have hanging out in an individual file at this stage.

Reply:
Gotcha. I'll yank for now.